Directory.Build.props

```diff
@@ -10,4 +28,9 @@
     <Target Name="DotnetToolRestore" BeforeTargets="Restore">
         <Exec Command="dotnet tool restore" />
     </Target>
+
+    <ItemGroup Condition=" '$(IsPublishable)' == 'True' ">
```

Comment:
Looks like this condition isn't properly ignoring most projects. I'm seeing the favicon and readme copied into most of the projects. I think we just need to explicitly add `<IsPublishable>false</IsPublishable>` to them.

Reply:
I think `IsPublishable` is true by default on all projects, I've moved this to `IsPackable`, which is false by default on test projects created with `dotnet new <test project flavour>`